sycl/include/sycl/accessor.hpp

```diff
@@ -2052,17 +2052,20 @@ class __SYCL_EBO __SYCL_SPECIAL_CLASS __SYCL_TYPE(accessor) accessor :
   bool empty() const noexcept { return size() == 0; }
 
   template <int Dims = Dimensions, typename = detail::enable_if_t<(Dims > 0)>>
-  range<Dimensions> get_range() const {
-    return detail::convertToArrayOfN<Dimensions, 1>(getAccessRange());
+  range<Dims> get_range() const {
+    if constexpr (Dimensions == 0) {
```

Comment:
As I know there is a rule to align if-else blocks like:
```
if (...) {
...
} else {
...
}
```
Or - without brackets at all

Reply:
thanks @KornevNikita,  that's right. not sure why I did it like that.